package.json

```diff
@@ -19,7 +19,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI=false react-scripts build",
```

Comment:
The project doesn't compile in production mode due to lint warnings around prop types. 
I created a ticket to remove this once we migrate everything over to Typescript